ynput/OpenPype
# Maya: Convert frame values to integers

## Changelog Description
Convert frame values to integers.

## Additional info
Float values are affecting few things, mainly burnin values where is added `.0` to the end of each string, which also causes that current frame has +2 `0` at the beginning of the value.

## Testing notes:
1. Make sure you have enabled burnins and one of the values contain `{frame_start}-{current_frame}-{frame_end}`
2. Make review in Maya
3. Publish
4. Validate look of burnin values (`1001-1001-1001` vs. `1001.0-001001-1001.0`)
